gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```

Comment:
gulp was being used for the server mode but jshint ignored the node mode comment above that line. that line being ignored making the server not run and build the dust unless the use strict line was specifically excluded